tests/performance_tests/performance_test.py

```diff
@@ -95,6 +97,8 @@ def performPtbReverseLinearSearch(tpsInitial: int, step: int, testHelperConfig:
     maxTpsReport = {}
     searchResults = []
     maxFound = False
+    if Utils.getNodeosVersion().split('.')[0] == "v2":
+        oldNodeos = True
```

Comment:
This check could be moved into `main()` and then `oldNodeos` could be set on `testClusterConfig`.